src/main/java/Deadlines.java

```diff
@@ -0,0 +1,27 @@
+public class Deadlines extends Task {
+    protected String taskType = "[D]";
+    protected String deadline;
+
+    public Deadlines(String description, String d) {
+        super(description);
+        this.deadline = d;
+    }
+
+    @Override
+    public void fullDesc() {
```

Comment:
```suggestion
    public void getFullDesc() {
```

It would be better to [use verbs to describe method names](https://se-education.org/guides/conventions/java/intermediate.html#naming)